setup.cfg

```diff
@@ -1,71 +0,0 @@
-[metadata]
-name = pysatMissions
```

Comment:
We probably want to keep this around for a bit.

Reply:
I need to run a test to see what happens if we maintain two sets of metadata.  I'm not sure if pip will grab the updated info if we have two.